<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,20">
      <item>
       <layout class="QHBoxLayout" name="infoLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="nickLabel">
          <property name="text">
           <string>Your Nick:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rigthsLabel">
          <property name="text">
           <string>Your rights:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="text">
           <string>Server:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="text">
         <string>/</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filesLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDownload"/>
   <addaction name="actionUpload"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionAboutProgram"/>
   <addaction name="actionAboutQt"/>
  </widget>
  <action name="actionUpload">
   <property name="icon">
    <iconset>
     <normaloff>../icons/upload.png</normaloff>../icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload your folders and files to server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset>
     <normaloff>../icons/download.png</normaloff>../icons/download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download selected folder and files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../icons/delete.png</normaloff>../icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected files and folders</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAboutProgram">
   <property name="icon">
    <iconset>
     <normaloff>../icons/info.png</normaloff>../icons/info.png</iconset>
   </property>
   <property name="text">
    <string>aboutProgram</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset>
     <normaloff>../icons/qt-logo.png</normaloff>../icons/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>aboutQt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
